Sources/PayPalWebPayments/PayPalVaultRequest.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+/// A request to vault a PayPal payment method
+public struct PayPalVaultRequest {
+
+    /// PayPal URL for the approval web page
+    public let url: URL
+
+    /// ID for the setup token associated with the vault
+    public let setupTokenID: String
+
+    public init(url: URL, setupTokenID: String) {
```

Comment:
Lets add some docstrings to this public init

Reply:
addressed here: bbc4e576f8908793cfbad11761cf66a28862ca2e